CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - Allow direct embedding vector upload together with dataset items. `DatasetItem` now has an additional parameter called `embedding_info` which can be used to directly upload embeddings when a dataset is uploaded.
+- Added `dataset.search_indexes` property, which exposes information about every search index which belongs to the dataset.   
```

Comment:
is it `embedding_indexes` now or `search_indexes`?

Reply:
should also be `embedding_indexes`